### ISSUE TITLE
Feature: Autorefresh on Inbox Page

### ISSUE THREAD

**Issue by SpikePy** [jhillyerd/inbucket#92]
We already have a autorefresh on the Monitor page but on the Inbox page you have to press refresh to see changes (if there is new mail). 

It would be awesome if we would have the same feature on the Inbox page of a user so that you don't have to press refresh to see if there is new stuff.
![image](https://user-images.githubusercontent.com/16565183/38452085-15461984-3a3d-11e8-9419-d636249e8745.png)

**Comment by jhillyerd** [comment]
Yeah, this is something I've been planning on adding - but the current web UI is a crusty hack with a sprinkling of jQuery.  After 2.0 release I will be rebuilding the UI (probably in Vue.js), and can add this in.

The backend websocket already supports this; so monitor a single mailbox: http://demo.inbucket.org/monitor/oliver - but definitely not as useful or convenient as having it on the regular mailbox page.

**Comment by somik123** [comment]
Was a autorefresh ever added or just manual refresh with button?

**Comment by jhillyerd** [comment]
Not yet added, still would like to have it.
